lms/djangoapps/teams/static/teams/js/spec/teams_tab_spec.js

```diff
@@ -1,80 +0,0 @@
-define([
```

Comment:
Any idea why this is marked as deleted -> added instead of moved like the other files?